src/main/java/store/domain/Promotion.java

```diff
@@ -0,0 +1,55 @@
+package store.domain;
+
+import static camp.nextstep.edu.missionutils.DateTimes.now;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class Promotion {
+    private final String name;
+    private final BigDecimal buyAmount;
+    private final BigDecimal getAmount;
+    private final LocalDate promotionStartDate;
+    private final LocalDate promotionEndDate;
+
+    public Promotion(String name, BigDecimal buyAmount, BigDecimal getAmount,
+                     LocalDate promotionStartDate, LocalDate promotionEndDate) {
+        this.name = name;
+        this.buyAmount = buyAmount;
+        this.getAmount = getAmount;
+        this.promotionStartDate = promotionStartDate;
+        this.promotionEndDate = promotionEndDate;
+    }
+
+
+    public String getName() {
+        return name;
+    }
+
+    public BigDecimal getBuyAmount() {
+        return buyAmount;
+    }
+
+    public BigDecimal getGetAmount() {
+        return getAmount;
+    }
+
+    public LocalDate getPromotionStartDate() {
+        return promotionStartDate;
+    }
+
+    public LocalDate getPromotionEndDate() {
+        return promotionEndDate;
+    }
+
+    public boolean isActive() {
+        LocalDate nowDate = now().toLocalDate();
+        return (promotionStartDate.isEqual(nowDate) || promotionStartDate.isBefore(nowDate)) && (
+                promotionEndDate.isEqual(nowDate) || promotionEndDate.isAfter(nowDate));
```

Comment:
`isNotActive()`로
```suggestion
        return promotionStartDate.isBefore(nowDate) || promotionEndDate.isAfter(nowDate);
```
이런식으로 간단하게 표현하는건 어떤가요?

Reply:
제가 작성 과정에서 실수가 있었는 것 같습니다.. 잘못된 로직 잡아주셔서 감사합니다!